npm_and_yarn/helpers/patches/npm++pacote+9.5.12.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/node_modules/npm/node_modules/pacote/lib/util/git.js b/node_modules/npm/node_modules/pacote/lib/util/git.js
+index 7642eb2..7bb3324 100644
+--- a/node_modules/npm/node_modules/pacote/lib/util/git.js
++++ b/node_modules/npm/node_modules/pacote/lib/util/git.js
+@@ -25,7 +25,8 @@ const GOOD_ENV_VARS = new Set([
+   'GIT_SSH',
+   'GIT_SSH_COMMAND',
+   'GIT_SSL_CAINFO',
+-  'GIT_SSL_NO_VERIFY'
++  'GIT_SSL_NO_VERIFY',
++  'GIT_CONFIG_GLOBAL'
```

Comment:
Really a shame upstream isn't more interested in this patch...

Reply:
Actually I did not propose this patch at all.

Apparently pacote now uses https://www.npmjs.com/package/@npmcli/git which I don't think resets the environment anymore.

So this is just temporary until we drop NPM6 support, which is the only thing relying on this old pacote.